.github/workflows/ci.yml

```diff
@@ -9,9 +9,6 @@ on:
       - '**.md'
       - 'LICENSE'
   pull_request:
-    paths-ignore:
```

Comment:
Drop push filters too, so that actions rerun when needed?

Reply:
Those are only for pushes to the main branch, so that should only take effect after the merge and won't affect PRs.